### PR TITLE
Created CODE_OF_CONDUCT.md and updated CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ashish-timilsina**
As discussed in the mailing list, I have copied ASF's CoC. 
@alficles please let me know if you have any questions/suggestions

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/549/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/504/

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ The following document is a set of guidelines to streamline the contribution pro
 
 Things to know before getting started
 -------------------------------------
+#### Code of Conduct
+This project and everyone participating in it is governed by the [Apache Software Foundation Code of Conduct](https://github.com/apache/incubator-trafficcontrol/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to adhere to this code.
```
Comment (alficles) [inline CONTRIBUTING.md]: On a second look, this sentence could use re-wording. The code of conduct is more of a collection of mutual goals and guidelines than something people could genuinely "adhere" to. Also, there's an awkward subject-verb agreement issue in the first clause. I still ok with it as it is, but if you wanted to change it, you might consider something like:

> Contributors and members of this project conduct themselves according to the [Apache Software Foundatation Code of Conduct](link). When you contribute or participate in the project, we expect that you'll you to do likewise.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ The following document is a set of guidelines to streamline the contribution pro
 
 Things to know before getting started
 -------------------------------------
+#### Code of Conduct
+Please try to keep discussions respectful and follow the [Apache Software Foundation Code of Conduct](https://github.com/apache/incubator-trafficcontrol/blob/master/CODE_OF_CONDUCT.md) when interacting with others.
```
Comment (alficles) [inline CONTRIBUTING.md]: This looks much better.

**Pull request merged**